packages/deploy/script/createFile.ts

```diff
@@ -30,7 +30,7 @@ async function run() {
   console.log("codeSize", codeSize);
 
   const tx = await client.writeContract({
-    address: deploys[chain.id].contracts.FileStore.address,
+    address: `0x${deploys[chain.id].contracts.FileStore.address.substring(2)}`,
```

Comment:
```suggestion
    address: deploys[chain.id].contracts.FileStore.address as Hex,
```

Reply:
actually the underlying issue is it's not finding the right TS file, will fix in main

Reply:
can you pull this in? https://github.com/holic/ethfs/commit/f7faf2fa86d95bd693691c26226f564a971be3a9